com.github.rhyme_player.rhyme.yml

```diff
@@ -0,0 +1,66 @@
+app-id: com.github.rhyme_player.rhyme
+runtime: org.freedesktop.Platform
+runtime-version: "20.08"
```

Comment:
21.08 is just out - could you try running with that instead? It'll obviously be in security support longer and should otherwise be newer and better.

Reply:
It was 21.08 but some one asked me to change it

Reply:
there was no BaseApp for 21.08 at the time, but the problem got addressed and there is a 21.08 baseapp now. So it's good to go.

The problem was the mix and match you did.